app/src/main/java/org/fossasia/pslab/communication/sensors/ADS1115.java

```diff
@@ -0,0 +1,238 @@
+/**
+ * Created by asitava1998 on 3/5/17.
+ */
+
```

Comment:
1st line should be of package. Reformat code please!
can remove created by info.